deploy/Chart/templates/_helpers.tpl

```diff
@@ -12,3 +12,32 @@
 {{- end -}}
 {{- print $version }}
 {{- end -}}
+
+{{/*
+Reuses the value from an existing secret, otherwise sets its value to a default value.
+
+Usage:
+{{ include "secrets.lookup" (dict "secret" "secret-name" "namespace" "ns-name" "key" "key-name" "defaultValue" "default-secret") }}
```

Comment:
nit: perhaps ...."defaultValue" "default-secretvalue") or ...."defaultValue" "default-value") is more accurate